data/category.go

```diff
@@ -116,17 +119,24 @@ func (c *Category) Seed(cfg write.ClientConfig) (int, error) {
 	}
 }
 
-func (c *Category) Test(cfg write.ClientConfig) error {
+func (c *Category) Test(cfg write.ClientConfig) (err error) {
```

Comment:
I was imagining that `err` was when something legitimately went wrong. I'll add some GoDoc to clarify though.